lib/assets/javascripts/turbograft/turbolinks.coffee

```diff
@@ -307,25 +368,9 @@ class window.Turbolinks
     validContent = ->
       xhr.getResponseHeader('Content-Type').match /^(?:text\/html|application\/xhtml\+xml|application\/xml)(?:;|$)/
 
-    extractTrackAssets = (doc) ->
-      for node in doc.querySelector('head').childNodes when node.getAttribute?('data-turbolinks-track')?
-        node.getAttribute('src') or node.getAttribute('href')
-
-    assetsChanged = (doc) ->
-      loadedAssets ||= extractTrackAssets document
```

Comment:
If `loadedAssets` is no longer being used as a cache, it can be removed from `window.Turbolinks`'s scope.